collectors/modality-probe-collector-common/src/lib.rs

```diff
@@ -640,8 +743,21 @@ impl TryFrom<&[u8]> for Report {
                                 .interpret_as_event_id()
                                 .ok_or_else(|| SerializationError::InvalidEventId(raw_entry))?,
                         );
+                    } else if raw_entry.has_wall_clock_time_bits_set() {
```

Comment:
Don't we need to check this branch first?  Since the other two branches check a subset?

Reply:
Ah, I see this is not needed now that the `has_clock_bit_set` and `has_event_with_payload_bit_set` methods have been updated to really mean `has_*_signature_in_reserved_bits`.